packages/cli/src/plugins/resource/plugin-standard-javascript.js

```diff
@@ -28,6 +31,40 @@ class StandardJavaScriptResource extends ResourceInterface {
       }
     });
   }
+
+  async shouldPreIntercept(url) {
+    return url.protocol === 'file:' && this.extensions.includes(url.pathname.split('.').pop());
```

Comment:
I think we only want to `preIntercept` if `polyfills.importAttributes` is true... could save a slight amount of processing here